google-oidc/app.py

```
@@ -47,18 +68,51 @@ def logout():
+    code_verifier = session.get("code_verifier")
+
+    token_data = {
+        "grant_type": "authorization_code",
+        "code": code,
+        "redirect_uri": url_for("callback", _external=True),
+        "client_id": app.config["GOOGLE_CLIENT_ID"],
+        "client_secret": app.config["GOOGLE_CLIENT_SECRET"],
+        "code_verifier": code_verifier
+    }
+
+    token_res = requests.post(token_endpoint, data=token_data).json()
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)